Makefile.am

```diff
@@ -113,6 +113,17 @@ dsm_inverse_SOURCES = bin/inverse.c
 
 dsm_lookup_SOURCES = bin/lookup.c
 
+if TESTS
+bin_PROGRAMS += all_tests
```

Comment:
Is the test scenario you're aiming for something like
* ../configure --enable-tests
* make
* ./tests

Or is make check considered?

To be honest I don't have a strong opinion on wether make check should be used when it's not actually running the tests, but this means writing a test harness for cmocka IIUC